.npmrc

```diff
@@ -1,2 +1,2 @@
-//npm.pkg.github.com/:_authToken=${NPM_TOKEN}
+//npm.pkg.github.com/:_authToken=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
```

Comment:
uhmm, that's not good :) 
Please revert this change and we need the actual variable to inject it at runtime...also rotate your secret please.